pkg/ociclient/upload.go

```diff
@@ -17,7 +17,7 @@ import (
 	v1 "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-const chunkSize = 100 * 1024 * 1024 // 100 MB chunks
+const chunkSize = 10 * 1024 * 1024 // 10 MB chunks
```

Comment:
this is just for testing, right?  10MB seems tiny.

Reply:
yeah, i think this needs to be a little larger (maybe 50 MB), but also more dynamic or user settable.  larger sizes are great for CI, but smaller will be good for poor client connections , right ?